src/sample.rs

```diff
@@ -0,0 +1,12 @@
+use std::ptr::null_mut;
```

Comment:
Shouldn't we put this in `examples/` or `test/` ?

Reply:
The samples is just here for testing, I'll prob put it in an example folder.

Reply:
Aye, its jsut to test the example.